packages/urls/src/urls.ts

```diff
@@ -1,167 +1,296 @@
+import { withPlatformUtm } from './platform-utm';
 import { Url } from './types';
 
```

Comment:
I found 3 URLs that are not defined in this file but somewhere else in suite-native folder:
```
https://trezor.io/support/troubleshooting/trezor-suite-issues/trezor-recovery-issues#lost-wallet-backup
https://trezor.io/learn/supported-assets/other-cryptocurrencies/destination-tags
https://trezor.io/support/troubleshooting/device-issues/how-to-enter-pin-on-model-one
```

Reply:
https://github.com/trezor/trezor-suite/pull/21108/commits/ac176b77922250177f7ee4d0ac02f3a2b39e5dcf 🙇‍♂️